gradle/posix.gradle

```diff
@@ -104,3 +104,22 @@ kotlin {
 }
 
 project.ext.set("hasNative", true)
+
+
+// hack for 1.4-M2, remove after
+// see https://youtrack.jetbrains.com/issue/KT-39302
+afterEvaluate {
```

Comment:
Is it still required? The ticket is marked as fixed

Reply:
There is `fixed on ktor-side` in ticket text, so let's wait for them to answer